### DISCUSSION TITLE
如何同时训练多个模型

### DISCUSSION THREAD

**Discussion by WuerLei** [fishaudio/Bert-VITS2#362] [Q&A]
比如我有10个人的音频，我想要同时训练这10个人，然后有10个模型文件，推理可以下拉选择人物

**Comment by GDbbq** [comment]
一人一个模型还是10个说话者的模型？  可以参考webui_preprocess

**Reply by WuerLei** [reply]
多谢回复，当然能一个模型包含10个说话人是最好了，主要是不想每个说话人都走一遍训练流程
